benchmarks/domains/hartmann/hartmann_3d.py

```diff
@@ -0,0 +1,172 @@
+"""Hartmann function in 3 dimensions as a benchmark."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+import pandas as pd
+from pandas import DataFrame
+
+from baybe.campaign import Campaign
+from baybe.parameters import NumericalContinuousParameter
+from baybe.recommenders import RandomRecommender
+from baybe.searchspace import SearchSpace
+from baybe.simulation import simulate_scenarios
+from baybe.targets import NumericalTarget
+from benchmarks.definition.convergence import (
+    ConvergenceBenchmark,
+    ConvergenceBenchmarkSettings,
+)
+
+if TYPE_CHECKING:
+    from mpl_toolkits.mplot3d import Axes3D
+
+
+def _hartmann_3d(x: np.ndarray) -> np.ndarray:
+    """Calculate the Hartmann function in 3D.
+
+    Args:
+        x: Input array of shape (n, 3) where n is the number of points
```

Comment:
In all our google docstrings, we end the sentences with full stop (see also other files added in the PR)